recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
     - setup.py.patch
 
 build:
-  number: 1002
+  number: 1000
```

Comment:
This can be set back to 0.

Reply:
1000 is only for same versions with new vs old compilers.

Reply:
Hi Matt! Do you know why it was 1002?

Reply:
Ah, I see. Thanks.

Reply:
This was an autogenerate build number when the compilers were being migrated in all of conda-forge. For packages at the same version, the ones built with new compilers were set to 1000 + X so they would be higher priority in the solver. Once a version is bumped, then that always has higher priority over a higher build number of a previous version, so we don't need that anymore.